packages/core/docs/integrations/redis-cache.md

```diff
@@ -40,9 +41,15 @@ export default {
 };
 ```
 
-We can break down package configuration into two pieces:
+We can break down package configuration into three pieces:
 
+* `enabled` (required boolean) - contains boolean value, that is responsible for the package's running status.
 * `invalidation` - please refer to [SSR Cache configuration](../performance/ssr-cache.md) page.
 * `driver` - object containing:
   * `defaultTimeout` - number of seconds until records expire, even if not invalidated;
-  * `redis` - object directly passed to [ioredis](https://github.com/luin/ioredis/blob/master/API.md#new-redisport-host-options);
\ No newline at end of file
+  * `redis` - object directly passed to [ioredis](https://github.com/luin/ioredis/blob/master/API.md#new-redisport-host-options);
+
+
+::: warning
+If you are using VSF cloud, please make sure that the redis host is set to `redis` for the production mode/ENV variable, otherwise, your VSF app will no connect to the Redis server!
```

Comment:
```suggestion
If you use the Vue Storefront Cloud, set the `driver.redis.host` property to `redis` in the production environment. It will be automatically mapped to the address of the Redis instance.
```